### PR TITLE
[README] File updated

### LINKED ISSUE #4: Update README.md
While going through <a href="https://github.com/amino19/Health-Tracker/blob/master/CONTRIBUTING.md"> CONTRIBUTING.md <a> 
- Enhance <a href="https://github.com/amino19/Health-Tracker/blob/master/README.md"> README.md </a> in designing way, so contributors shall get an idea, what's going here.
- Add some badges to looks trendy!

**Tech Stack reqd.** : HTML, Badges digger, Technical English, Documentation

--------
**Resources** 
- <a href="https://dev.to/envoy_/150-badges-for-github-pnk"> Badges </a>
- Prefer any organization `README file` to make it more cool!

### PR DESCRIPTION
**Pull request by j-dogcoder**

### CODE REVIEW

**Comment by j-dogcoder** [comment]
Closes #4

File: README.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
-```
-npm run serve
-```
+* Github: [@amino19](https://github.com/amino19)
 
-### Compiles and minifies for production
-```
-npm run build
-```
+## Show your support
```
Comment (amino19) [inline README.md:27]: @j-dogcoder : Can you pl put this section at bottom? :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
+
 ![image](https://user-images.githubusercontent.com/75872316/120891093-442f3980-c624-11eb-8392-29032c38bbda.png)
 
+## Author
 
-## Project setup
-```
-npm install
-```
+👤 **amino19**
```
Comment (amino19) [inline README.md:23]: @j-dogcoder : Pl add <a href="https://github.com/amino19">that</a> link here, & remove the unorder-point from below! :slightly_smiling_face:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,42 +2,46 @@
 
 Please do! Thanks for your help in improving the project! :balloon:
 
-All contributors are welcome. Not sure where to start? Please do simulate this project from below details given, so to get an idea of Project's workflow.
-All set to contribute? Grab an open issue, with permission to <a href="https://github.com/amino19"> mine 🙂 </a> and hop in. Join our <a href="https://github.com/amino19/Health-Tracker/discussions/"> discussions</a> and engage in conversation. Create a [new issue](/../../issues/new/choose) if needed. All [pull requests](/../../pulls) should ideally reference an open [issue](/../../issues). Include keywords in your pull request descriptions, as well as commit messages, to automatically close related issues in GitHub.
+All contributors are welcome. Not sure where to start? Please follow the directions from the details given below, so you can get an idea of the project's workflow.
```
Comment (amino19) [inline CONTRIBUTING.md:5]: Pl don't mind to change this line as:
```
All contributors are welcome. Not sure where to start? Please follow the directions to simulate from the details given below, so you can get an idea of the project's workflow.
```

File: README.md [hunk 4]
```diff
@@ -1,42 +1,43 @@
...
 
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
+* Github: [@amino19](https://github.com/amino19)
```
Comment (amino19) [inline README.md:25]: @j-dogcoder : Pl remove this line, as it will be going to covered with `Line 23` :rocket:

**Review by amino19 — CHANGES_REQUESTED** [review]

**Comment by amino19** [comment]
@j-dogcoder : Almost done! :partying_face:

File: CONTRIBUTING.md (hunk 3 above)
Comment (j-dogcoder) [reply CONTRIBUTING.md:5]: Can you please clarify a bit @amino19?

Comment (amino19) [reply CONTRIBUTING.md:5]: @j-dogcoder : Copy paste the line in place of `Line 5` 🙂

File: README.md [hunk 5]
```diff
@@ -1,42 +1,41 @@
...
-![image](https://user-images.githubusercontent.com/75872316/120891023-f4507280-c623-11eb-91b9-718e4dd22ea9.png)
+## Project setup
 
+See the [contributing.md](https://github.com/amino19/Health-Tracker/blob/master/CONTRIBUTING.md) file for more details on how to setup.
 
 ### Customize configuration
+
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+## Show your support
```
Comment (amino19) [inline README.md:39]: @j-dogcoder : LGTM! 🎉

**Review by amino19 — CHANGES_REQUESTED** [review]

**Comment by amino19** [comment]
@j-dogcoder : Thanks Jasper for the hardwork! 
You made it somehow! :partying_face:

**Comment by j-dogcoder** [comment]
🥳

**Comment by j-dogcoder** [comment]
Anything else you need @amino19?

**Comment by j-dogcoder** [comment]
I am going to install the project and give it a try.

**Comment by amino19** [comment]
@j-dogcoder : Actually it's not functional yet!

**Comment by j-dogcoder** [comment]
Oh! You will have to let me know when it is!

**Comment by amino19** [comment]
@j-dogcoder : Do you mind to share me your contact panel, like via text social platform?

**Comment by j-dogcoder** [comment]
My discord is J-dogcoder#0803

**Comment by amino19** [comment]
@j-dogcoder : Just sent you from `aminos 🇮🇳#2567` :slightly_smiling_face:

**Pull request merged**